requirements_test.txt

```
@@ -1,4 +1,4 @@
-dr-usertool==0.5.18
+dr-usertool>=1.0.0
```

```suggestion
dr-usertool>=1.0.3,<2
```
semantic version to avoid breaking changes planned in 2.0